HandleNHCreateOrUpdateInstallationCallOrchestrator/handler.ts

```diff
@@ -0,0 +1,50 @@
+import { Task } from "durable-functions/lib/src/classes";
+import * as t from "io-ts";
+import * as o from "../utils/durable/orchestrators";
+
+import { CreateOrUpdateInstallationMessage } from "../generated/notifications/CreateOrUpdateInstallationMessage";
+import { ActivityBodyImpl as CreateOrUpdateActivityBodyImpl } from "../HandleNHCreateOrUpdateInstallationCallActivity";
+import { NotificationHubConfig } from "../utils/notificationhubServicePartition";
+
+export const OrchestratorName =
+  "HandleNHCreateOrUpdateInstallationCallOrchestrator";
+
+/**
+ * Carries information about Notification Hub Message payload
+ */
+export const NhCreateOrUpdateInstallationOrchestratorCallInput = t.interface({
+  message: CreateOrUpdateInstallationMessage
+});
+
+export type NhCreateOrUpdateInstallationOrchestratorCallInput = t.TypeOf<
+  typeof NhCreateOrUpdateInstallationOrchestratorCallInput
+>;
+
+interface IHandlerParams {
+  createOrUpdateActivity: o.CallableActivity<CreateOrUpdateActivityBodyImpl>;
+  notificationHubConfig: NotificationHubConfig;
+}
+
+export const getHandler = ({
+  createOrUpdateActivity,
+  notificationHubConfig
+}: IHandlerParams) => {
+  return o.createOrchestrator(
```

Comment:
`o.createOrchestrator` it's a builder function that takes an OrchestratorBody (aka its business logic) and wraps it with input validation and error handling